2020-01-Winter/1_linked_lists/list-palindrome/solution.java

```diff
@@ -0,0 +1,41 @@
+import java.util.*;
+
+/*
+A simple solution is to use a stack of list nodes. This mainly involves three steps.
+Traverse the given list from head to tail and push every visited node to stack.
+Traverse the list again. For every visited node, pop a node from stack and 
+compare data of popped node with currently visited node.
+If all nodes matched, then return true, else false.
+ */
+
+public class solution {
+    static boolean isPalindrome(Node head) 
```

Comment:
As given, the code for this file won't compile because `Node` is not defined in this class or as a public class. Also, if trying to compile `driver.java` separately, an error is thrown because the solution method is empty and is therefore missing a boolean return statement. 

To fix this, you can add your solution method here to `driver.java` and remove this file.